### PR TITLE
Hannah Jesinkey API

### PR DESCRIPTION
**Pull request by jesinkey**
https://mongoapi-hannahj.herokuapp.com/movies

### CODE REVIEW

**Comment by jesinkey** [comment]
https://mongoapi-hannahj.herokuapp.com/movies

File: server.js [hunk 1]
```diff
@@ -1,39 +1,96 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
```
Comment (matildasvensson85) [inline server.js:6]: Remove unused code :)

**Review by matildasvensson85 — APPROVED** [review]
Great project, nice and clean code. Just remove some unused code :)

File: server.js [hunk 2]
```diff
@@ -1,39 +1,88 @@
...
+const ReleaseYear = mongoose.model('ReleaseYear', {
+  release_year: Number,
+});
+
+const Movie = mongoose.model('Movie', {
+  title: String,
+  director: String,
+  release_year: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'ReleaseYear',
```
Comment (KarolCislowski) [inline server.js:23]: 😻

File: server.js [hunk 3]
```diff
@@ -1,39 +1,88 @@
...
-  res.send('Hello world')
-})
+app.get('/movies', async (req, res) => {
+  const movies = await Movie.find().populate('release_year');
+  res.json(movies);
+});
+
+app.get('/movies/:id', async (req, res) => {
+  const movies = await await Movie.findById(req.params.id);
+  console.log(movies);
```
Comment (KarolCislowski) [inline server.js:57]: remember to clean your code 🧹

File: server.js [hunk 4]
```diff
@@ -1,39 +1,88 @@
...
+  if (movies) {
+    res.json(movies);
+  } else {
+    res.status(404).json({ error: 'Movie Not Found' });
```
Comment (KarolCislowski) [inline server.js:61]: Nice error message

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**